app/src/main/java/com/gh4a/activities/PullRequestActivity.java

```diff
@@ -520,6 +573,86 @@ private void handlePullRequestUpdate() {
         supportInvalidateOptionsMenu();
     }
 
+    private class RestoreBranchTask extends ProgressDialogTask<Reference> {
+        public RestoreBranchTask() {
+            super(getBaseActivity(), R.string.saving_msg);
+        }
+
+        @Override
+        protected ProgressDialogTask<Reference> clone() {
+            return new RestoreBranchTask();
+        }
+
+        @Override
+        protected Reference run() throws Exception {
+            DataService dataService =
+                    (DataService) Gh4Application.get().getService(Gh4Application.DATA_SERVICE);
+
+            PullRequestMarker head = mPullRequest.getHead();
+            String owner = head.getRepo().getOwner().getLogin();
```

Comment:
getOwner() (and/or getRepo()?) might return null if the user left Github since the PR creation. Not sure how to test that though :-/

Reply:
We could hide this action from the menu during onCreateOptionsMenu it if these are null. Would that work?